<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Recording Settings</string>
  </property>
  <widget class="Backdrop" name="backdrop">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>551</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Preset</string>
   </property>
  </widget>
  <widget class="ComboBox" name="uiPresets">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>381</width>
     <height>81</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>%i×%i @ %ifps</string>
    </property>
   </item>
  </widget>
  <widget class="SpinBox" name="uiHRes">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>SpinBox::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiVRes">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>SpinBox::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiHOffset">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="SpinBox" name="uiVOffset">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiMaximizeFramerate">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Label" name="label">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>250</y>
     <width>30</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>fps</string>
   </property>
  </widget>
  <widget class="Label" name="label_2">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>300</y>
     <width>140</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>frame duration</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>10</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid black; border-right-width: 0px;</string>
   </property>
  </widget>
  <widget class="SpinBox" name="uiExposure">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>221</width>
     <height>61</height>
    </rect>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>SpinBox::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiMaximizeExposure">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>330</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Offset</string>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame Rate</string>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exposure</string>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Analog Gain</string>
   </property>
  </widget>
  <widget class="Button" name="uiDone">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>400</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::half</enum>
   </property>
  </widget>
  <widget class="Button" name="uiCenterRecording">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>250</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Recenter</string>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop_2">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>251</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop_3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="Backdrop" name="backdrop_4">
   <property name="geometry">
    <rect>
     <x>781</x>
     <y>20</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="uiPreviewPanel" native="true">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>70</y>
     <width>231</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgba(0,0,0,255)</string>
   </property>
   <widget class="QWidget" name="uiPassepartoutTop" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>1</y>
      <width>229</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; background: rgba(255,255,255,128)</string>
    </property>
   </widget>
   <widget class="QWidget" name="uiPassepartoutBottom" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>141</y>
      <width>229</width>
      <height>39</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; background: rgba(255,255,255,128)</string>
    </property>
   </widget>
   <widget class="QWidget" name="uiPassepartoutLeft" native="true">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>40</y>
      <width>39</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; background: rgba(255,255,255,128)</string>
    </property>
   </widget>
   <widget class="QWidget" name="uiPassepartoutRight" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>40</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none; background: rgba(255,255,255,128)</string>
    </property>
   </widget>
   <widget class="QWidget" name="uiPassepartoutInnerBorder" native="true">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>39</y>
      <width>151</width>
      <height>102</height>
     </rect>
    </property>
   </widget>
   <zorder>uiPassepartoutLeft</zorder>
   <zorder>uiPassepartoutRight</zorder>
   <zorder>uiPassepartoutTop</zorder>
   <zorder>uiPassepartoutBottom</zorder>
   <zorder>uiPassepartoutInnerBorder</zorder>
  </widget>
  <widget class="Label" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Label" name="label_4">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Subheader" name="headerlabel_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Resolution</string>
   </property>
  </widget>
  <widget class="DecimalSpinBox" name="uiFrameDuration">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="suffix">
    <string>µs</string>
   </property>
   <property name="maximum">
    <double>1000000000000000043845843045076197354634047651840.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>DecimalSpinBox::none</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>DecimalSpinBox::half</enum>
   </property>
  </widget>
  <widget class="DecimalSpinBox" name="uiFps">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>231</width>
     <height>61</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>DecimalSpinBox::half</enum>
   </property>
   <property name="clickMarginBottom" stdset="0">
    <enum>DecimalSpinBox::half</enum>
   </property>
  </widget>
  <widget class="ComboBox" name="uiAnalogGain">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>400</y>
     <width>221</width>
     <height>71</height>
    </rect>
   </property>
   <property name="clickMarginTop" stdset="0">
    <enum>ComboBox::half</enum>
   </property>
   <item>
    <property name="text">
     <string>{dB}dB (×{multiplier})</string>
    </property>
   </item>
  </widget>
  <widget class="Label" name="uiUnsavedChangesWarning">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>400</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>* Press Done to apply resolution.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="HeaderLabel" name="headerlabel_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Recording Settings</string>
   </property>
  </widget>
  <widget class="Button" name="uiSavePreset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>💾</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::half</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
  </widget>
  <widget class="Button" name="uiDeletePreset">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>40</y>
     <width>51</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>×</string>
   </property>
   <property name="clickMarginLeft" stdset="0">
    <enum>Button::half</enum>
   </property>
   <property name="clickMarginRight" stdset="0">
    <enum>Button::none</enum>
   </property>
   <property name="customStyleSheet" stdset="0">
    <string>Button { color: maroon }</string>
   </property>
  </widget>
  <zorder>backdrop</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>backdrop_4</zorder>
  <zorder>backdrop_3</zorder>
  <zorder>backdrop_2</zorder>
  <zorder>headerlabel_2</zorder>
  <zorder>headerlabel</zorder>
  <zorder>headerlabel_4</zorder>
  <zorder>headerlabel_3</zorder>
  <zorder>headerlabel_5</zorder>
  <zorder>uiPresets</zorder>
  <zorder>uiHRes</zorder>
  <zorder>uiVRes</zorder>
  <zorder>uiHOffset</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>widget</zorder>
  <zorder>uiExposure</zorder>
  <zorder>uiCenterRecording</zorder>
  <zorder>uiPreviewPanel</zorder>
  <zorder>uiMaximizeFramerate</zorder>
  <zorder>uiVOffset</zorder>
  <zorder>headerlabel_6</zorder>
  <zorder>uiFps</zorder>
  <zorder>uiMaximizeExposure</zorder>
  <zorder>uiFrameDuration</zorder>
  <zorder>uiAnalogGain</zorder>
  <zorder>uiUnsavedChangesWarning</zorder>
  <zorder>uiDone</zorder>
  <zorder>headerlabel_7</zorder>
  <zorder>uiSavePreset</zorder>
  <zorder>uiDeletePreset</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Backdrop</class>
   <extends>QLabel</extends>
   <header>backdrop</header>
  </customwidget>
  <customwidget>
   <class>Button</class>
   <extends>QPushButton</extends>
   <header>button</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combo_box</header>
  </customwidget>
  <customwidget>
   <class>DecimalSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>decimal_spin_box</header>
  </customwidget>
  <customwidget>
   <class>HeaderLabel</class>
   <extends>QLabel</extends>
   <header>header_label</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spin_box</header>
  </customwidget>
  <customwidget>
   <class>Subheader</class>
   <extends>QLabel</extends>
   <header>subheader</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiDone</tabstop>
  <tabstop>uiPresets</tabstop>
  <tabstop>uiHRes</tabstop>
  <tabstop>uiVRes</tabstop>
  <tabstop>uiHOffset</tabstop>
  <tabstop>uiVOffset</tabstop>
  <tabstop>uiFps</tabstop>
  <tabstop>uiMaximizeFramerate</tabstop>
  <tabstop>uiCenterRecording</tabstop>
  <tabstop>uiFrameDuration</tabstop>
  <tabstop>uiExposure</tabstop>
  <tabstop>uiMaximizeExposure</tabstop>
  <tabstop>uiAnalogGain</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
